### ISSUE TITLE
Filtering Knowledge by name in processing (Automation/playbook)

### ISSUE THREAD

**Issue by ghost** [LuatixHQ/opencti#6062]
## Use case
I want to postprocess some entities before ingestion and drop a specific entity type with a specific name.

## Proposed Solution
Allow to filter by name

**Comment by nino-filigran** [comment]
@iFrozenPhoenix can you be more explicit: what do you mean by drop, to not ingest a specific entity? 
Can you explain your use case with more details?

**Comment by ghost** [comment]
There are some cases where Threat Actors (Intrusion Sets) are added as Malware in the reports. With such a solution they could be filtered out.

**Comment by nino-filigran** [comment]
So, in essence, the root cause is that some Threat Actors (or intrusion sets?) are added as the wrong entity type. Is it due to a specific specific way of ingesting data (RSS feed, connector...), that misinterprets the entity type? 

I'm not saying that your feature request is a wrong one, but simply trying to get to the bottom of your problem, to ensure that the solution we take is the proper one.

**Comment by ghost** [comment]
Yes, the main problem is that sometimes the intrusion sets are also added as malware. This mostly happens with the Alienvault connector. I think the root cause is how the entities are added in Alienvault, probably they are really added by mistake.
So due to the fact that 99% of the ingestions of this connector are correct, I think the reports aren't, therefore the idea was to cleanup these few cases.

**Comment by nino-filigran** [comment]
Ok fully understood thank you very much. We need to review slightly how automation works (to enhance it with additional uses cases and this one is definitively valid), so I'll update the ticket when we'll have a common approach for all of them. Thanks for taking the time to explain.

**Comment by nino-filigran** [comment]
Related to the data qanity use case: https://github.com/OpenCTI-Platform/opencti/issues/3571 (where you need a functionality to transform a type into another & to be able to pre-process entities before they enter in the platform)

**Comment by nino-filigran** [comment]
On this specific request, given that you can filter on "representative", filter on name is possible. However, the overal use case (of changing an entity into another type) is not met yet. I'll still close this ticket to highlight the fact that the requirement is met.
